.github/workflows/android-unit-tests.yml

```diff
@@ -41,51 +45,51 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '11'
-      - name: Get yarn cache directory path
+      - name: ♻️ Get yarn cache directory path
         id: yarn-cache-dir-path
         run: echo "::set-output name=dir::$(yarn cache dir)"
-      - uses: actions/cache@v2
-        name: Restore yarn cache
+      - name: ♻️ Restore yarn cache
+        uses: actions/cache@v2
```

Comment:
It's not the best way to cache yarn's cache — see other workflows that cache `node_modules` directories instead. Using two different caching approaches is problematic as we have limited storage for cache.